### ISSUE TITLE
Code samples use auto-generated examples instead 

### ISSUE THREAD

**Issue by simondean** [Mermade/widdershins#112]
It looks like widdershins uses an auto-generated request body in the code samples it generates, even if an OpenAPI v3 operation contains a request body example.   

Here's an example:
* OpenAPI definition: https://github.com/simondean/widdershins-issues/blob/master/request-examples/openapi.yaml#L31
* Generated HTML: https://simondean.github.io/widdershins-issues/request-examples/index.html#getgreeting

# Actual Output

In the actual output, the `greeting` field is automatically set to `string`

```
Body parameter

{
  "greeting": "string"
}
```

# Expected Output

In the example in the https://github.com/simondean/widdershins-issues/blob/master/request-examples/openapi.yaml file, the `greeting` field is explicitly set to `Request!` and it'd been good if that's what was rendered by widdershins.  

```
Body parameter

{
  "greeting": "Request!"
}
```

**Comment by MikeRalphson** [comment]
Should be fixed in v3.2.3 - please chase if not.

**Comment by simondean** [comment]
Thanks for the fix.  It's working well.
